eslint.config.js

```diff
@@ -0,0 +1,316 @@
+import jqueryConfig from "eslint-config-jquery";
+import importPlugin from "eslint-plugin-import";
+import globals from "globals";
+
+export default [
+	{
+
+		// Only global ignores will bypass the parser
+		// and avoid JS parsing errors
+		// See https://github.com/eslint/eslint/discussions/17412
+		ignores: [
+			"test/data/json_obj.js"
+		]
+	},
+
+	{
+		files: [
+			"eslint.config.js",
+			"Gruntfile.cjs",
+			"test/node_smoke_tests/commonjs/**",
+			"test/node_smoke_tests/module/**",
+			"test/promises_aplus_adapters/**",
+			"test/middleware-mockserver.cjs"
+		],
+		languageOptions: {
+			globals: {
+				...globals.node,
+				...globals.es2021
```

Comment:
Can we do that? We still run CI on Node 10 on our rusty Jenkins...

This applies to multiple places.

Reply:
I can at least confirm that the build works and the tests pass on Node 10. If we need to fix any issues on the Jenkins server, perhaps we can address that after the merge.

Reply:
We may not have issues now simply because we don't use any of these new globals. If we did, it would surely break on Jenkins. Can we lower that to `es2018` for now? That's what Node 10 supports.

Reply:
`es2018` is not on the of the options. There's `es2015`, `es2017`, and `es2020`. But removing this line actually works here.